migration/migrating-4-4/migrating-openshift-4_2-to-4.adoc

```diff
@@ -0,0 +1,71 @@
+[id="migrating-openshift-4_2-to-4"]
+= Migrating {product-title} 4.2 to 4.2
+include::modules/common-attributes.adoc[]
+:context: ocp-4_2-4_2
+:ocp-4_2-4_2:
+
+toc::[]
+
+You can migrate application workloads from {product-title} 4.2 to 4.2 with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
```

Comment:
same comment here, instead of saying 4.2 to 4.2, suggest "between {product-title} 4.2 clusters".